apps/dashboard/app/controllers/files_controller.rb

```diff
@@ -190,22 +187,6 @@ def uppy_upload_path
   end
 
   def show_file
-    type = Files.mime_type_by_extension(@path).presence || PosixFile.new(@path).mime_type
```

Comment:
I feel like we should follow the premise that duplication is better than the wrong abstraction. I feel like refactoring an ugly version that works is better than assuming and building abstractions before we need them.

Reply:
Thanks, it does seem like that would be a more reasonable approach after all. I'll change it once I have some time.

Reply:
Removed that part. I will refactor it later according to your advice once I submit the PR for remote files as it would be a quite small change with only posix files existing currently.
Though I could refactor to `send_posix_file` if you'd like that already, e.g. something like:
```diff
   def show_file
+    send_posix_file
+  end
+
+  def send_posix_file
     type = Files.mime_type_by_extension(@path).presence || PosixFile.new(@path).mime_type
```

Reply:
> Though I could refactor to send_posix_file if you'd like that already, e.g. something like:

I'm sort of ambivalent. Happy to pull this as is or with this slight change.

Reply:
Feel free to pull it as is.